lightyear/src/client/connection.rs

```diff
@@ -423,7 +423,7 @@ impl<P: Protocol> ReplicationSend<P> for ConnectionManager<P> {
         target: NetworkTarget,
         system_current_tick: BevyTick,
     ) -> Result<()> {
-        // trace!(?entity, "Send entity spawn for tick {:?}", self.tick());
+        info!(?entity, "Send entity spawn to server");
```

Comment:
This should probably still be a trace log

Reply:
Yes, the PR is still WIP :) 
Sorry I should have made it clearer